docs/DeveloperGuide.md

```diff
@@ -5,7 +5,13 @@ title: Developer Guide
 * Table of Contents
 {:toc}
 
-<!-- -------------------------------------------------------------------------------------------------------------- -->
+------------------------------------------------------------------------------------------------------------------------
```

Comment:
Can you copy the introduction from the UG? I have added a new introduction there.

Reply:
Doesn't it feel a bit too cheerful and promotional for a DG? Since DG is catered towards developers and "investors".